src/code/z_camera.c

```diff
@@ -2693,12 +2802,12 @@ s32 Camera_Jump3(Camera* camera) {
         eyeDiffGeo.pitch = Camera_CalcDefaultPitch(camera, eyeNextAtOffset.pitch, roData->pitchTarget, 0);
     }
 
-    if (eyeDiffGeo.pitch > R_CAM_MAX_PITCH) {
-        eyeDiffGeo.pitch = R_CAM_MAX_PITCH;
+    if (eyeDiffGeo.pitch > CAM_MAX_PITCH) {
+        eyeDiffGeo.pitch = CAM_MAX_PITCH;
     }
 
-    if (eyeDiffGeo.pitch < R_CAM_MIN_PITCH_1) {
-        eyeDiffGeo.pitch = R_CAM_MIN_PITCH_1;
+    if (eyeDiffGeo.pitch < CAM_MIN_PITCH_1) {
+        eyeDiffGeo.pitch = CAM_MIN_PITCH_1;
     }
```

Comment:
they can potentially use clamp even. but its probably out of scope for this PR, since its just trying to match things for retail. this seems more of a general improvement that can be researched separately

Reply:
yeah true, I was concerned by the size of the file but it's fine for now